frontend/src/components/RegisterBox.tsx

```diff
@@ -42,6 +17,8 @@ function RegisterBox() {
   });
 
   const [submitError, setSubmitError] = useState<string>("");
+  const [profilePictureBase64, setProfilePictureBase64] = useState<string>("");
+  const [pictureFile, setPictureFile] = useState<File>();
```

Comment:
Basic form input and error handling should be done in the user schema and the final conversion to base64 could be done in the submit function